astropy/io/fits/tests/test_header.py

```diff
@@ -50,23 +50,7 @@ class TestOldApiHeaderFunctions(FitsTestCase):
     This tests backward compatibility support for those interfaces.
     """
 
-    @ignore_warnings(AstropyDeprecationWarning)
-    def test_ascardimage_verifies_the_comment_string_to_be_ascii_text(self):
-        # the ascardimage() verifies the comment string to be ASCII text
-        c = fits.Card.fromstring('abc     = +  2.1   e + 12 / abcde\0')
-        pytest.raises(Exception, c.ascardimage)
-
-    @ignore_warnings(AstropyDeprecationWarning)
-    def test_rename_key(self):
-        """Test backwards compatibility support for Header.rename_key()"""
-        header = fits.Header([('A', 'B', 'C'), ('D', 'E', 'F')])
-        header.rename_key('A', 'B')
-        assert 'A' not in header
-        assert 'B' in header
-        assert header[0] == 'B'
-        assert header['B'] == 'B'
-        assert header.comments['B'] == 'C'
-
+    # FIXME: This one still works and does not trigger any warning
```

Comment:
This test is the last one in the `TestOldApiHeaderFunctions` class. It does not trigger deprecation warning, and seems to be the only test for `add_history`. So I think I will move it to the `TestHeaderFunctions` class.